packages/plugin-core/src/showcase/GraphThemeTip.ts

```diff
@@ -0,0 +1,45 @@
+import { assertUnreachable } from "@dendronhq/common-all";
+import { ShowcaseEntry } from "@dendronhq/engine-server";
+import * as vscode from "vscode";
+import {
+  DisplayLocation,
+  IFeatureShowcaseMessage,
+} from "./IFeatureShowcaseMessage";
+
+export class GraphThemeTip implements IFeatureShowcaseMessage {
+  shouldShow(displayLocation: DisplayLocation): boolean {
+    switch (displayLocation) {
+      case DisplayLocation.InformationMessage:
+      case DisplayLocation.TipOfTheDayView:
+        return true;
+      default:
+        assertUnreachable(displayLocation);
+    }
+  }
+  get showcaseEntry(): ShowcaseEntry {
+    return ShowcaseEntry.GraphTheme;
+  }
+
+  getDisplayMessage(displayLocation: DisplayLocation): string {
+    switch (displayLocation) {
+      case DisplayLocation.InformationMessage:
+        return `New themes for Graph View! Change the appearance...`;
```

Comment:
Oops - sorry miscommunication here.  I meant for the string to be

"New themes for Graph View! Change the appearance of the note graph by clicking the config button on the top left corner of the graph view and selecting one of the built-in styles. You can even customize the appearance to your liking with css"